spanner/tests/system/test_system.py

```diff
@@ -580,6 +581,30 @@ def test_read_w_index(self):
             [(row[0], row[2]) for row in self._row_data(ROW_COUNT)]))
         self._check_row_data(rows, expected)
 
+    def test_read_w_single_key(self):
```

Comment:
Worth factoring out a helper that takes a list of keys that you just dump into `KeySet`?